modules/backing-up-red-hat-quay.adoc

```diff
@@ -129,6 +181,15 @@ quayregistry-quay-database
 $ oc exec quayregistry-quay-database-59f54bb7-58xs7 -- /usr/bin/pg_dump -C quayregistry-quay-database  > backup.sql
 ----
 
+=== {productname} managed object storage backup
+
+The following instructions apply to both configurations, standalone multi-cloud object gateway and OpenShift Data Foundations storage that the {productname} Operator provisioned a S3 object storage bucket from, via the `ObjectStorageBucketClaim` API.
```

Comment:
This sentence is long and confusing. I'm not sure how to revise but here's an attempt: 

The instructions in this section apply to the following configurations: 

* Standalone, multi-cloud object gateway configurations
* OpenShift Data Foundations storage conditions that the {productname} Operator provisioned an S3 object storage bucket from, through the `ObjectStorageBucketClaim` API